src/system.cr

```diff
@@ -18,4 +18,13 @@ module System
       {len, len}
     end
   end
+
+  # Returns number of online CPUs
```

Comment:
May I suggest that we further detail what this method does? The current explanation will force anyone who doesn't know the exact meaning of "online CPUs" to review the `LibC` documentation. Ideally it'd be nice if we can save them the leap. I checked the docs in some other langs and they seem to use something like "number of [CPUs | Cores | Processors] available to the [program | process | runtime]"